deployer/models/service.go

```diff
@@ -505,6 +512,10 @@ func (service *Service) createLaunchConfigurationInput() *lc.LaunchConfigInput {
 
 	input.SpotPrice = service.SpotPrice
 
+	if service.PlacementTenancy != nil {
```

Comment:
you don't need to check nil, because it will just set the value as nil

Reply:
Fised